objectivec/GPBMessage.m

```diff
@@ -1808,17 +1812,20 @@ - (NSArray *)extensionsCurrentlySet {
 }
 
 - (void)writeExtensionsToCodedOutputStream:(GPBCodedOutputStream *)output
-                                     range:(GPBExtensionRange)range {
-  NSArray *sortedExtensions = [[extensionMap_ allKeys]
-      sortedArrayUsingSelector:@selector(compareByFieldNumber:)];
+                                     range:(GPBExtensionRange)range
+                                extensions: sortedExtensions {
```

Comment:
maybe the name should be "sortedExtensions", and you forgot to add a type (I'm guessing Objc is defaulting to id, although I had never realized this was potentially valid syntax).

Reply:
Done